lsp/src/saros/lsp/extensions/client/ISarosLanguageClientAware.java

```diff
@@ -0,0 +1,12 @@
+package saros.lsp.extensions.client;
+
+/** Interface for Saros language client awareness. */
+public interface ISarosLanguageClientAware {
```

Comment:
I am not sure why you need this interface. Can you explain the idea why you introduced it? Is the concept of being client aware something we can and should separate from being a language server?

Reply:
It is already separated in the framework of LSP4J so I continued their structure. Other than that a language server by itself doesn't need to be aware of the client, only if it needs to interact with the client out of the lsp specification.